### PR TITLE
Added Hashicorp Consul cookbook

### PR DESCRIPTION
**Pull request by RajeshwariMakond**
Added Hashicorp Consul cookbook

### CODE REVIEW

File: hconsul/metadata.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+name 'hconsul'
+maintainer 'The Authors'
```
Comment (aashishmodak) [inline hconsul/metadata.rb:2]: Add your name as maintainer

File: hconsul/metadata.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+maintainer_email 'you@example.com'
```
Comment (aashishmodak) [inline hconsul/metadata.rb:3]: Add your email id

File: hconsul/metadata.rb [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+description 'Installs/Configures hconsul'
+long_description 'Installs/Configures hconsul'
+version '0.1.1'
+chef_version '>= 12.1' if respond_to?(:chef_version)
+
+# The `issues_url` points to the location where issues for this cookbook are
+# tracked.  A `View Issues` link will be displayed on this cookbook's page when
+# uploaded to a Supermarket.
+#
+# issues_url 'https://github.com/<insert_org_here>/hconsul/issues'
```
Comment (aashishmodak) [inline hconsul/metadata.rb]: update issue url

File: hconsul/metadata.rb [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+# The `source_url` points to the development repository for this cookbook.  A
+# `View Source` link will be displayed on this cookbook's page when uploaded to
+# a Supermarket.
+#
+# source_url 'https://github.com/<insert_org_here>/hconsul'
```
Comment (aashishmodak) [inline hconsul/metadata.rb]: update source url

**Comment by aashishmodak** [comment]
Why do you need this file "hconsul/files/default/consul_0.9.0_linux_amd64/consul" ?

File: hconsul/README.md [hunk 5]
```diff
@@ -0,0 +1,4 @@
+# hconsul
+
+TODO: Enter the cookbook description here.
```
Comment (aashishmodak) [inline hconsul/README.md:3]: It says description is TODO?

Comment (aashishmodak) [reply hconsul/README.md:3]: refer readme from https://github.com/miketheman/nginx/tree/v2.6.0

File: hconsul/metadata.rb [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+description 'Install/Configures hconsul'
+long_description 'Install/Configures hconsul'
+version '0.1.1'
+chef_version '>= 12.1' if respond_to?(:chef_version)
+
+# The `issues_url` points to the location where issues for this cookbook are
+# tracked.  A `View Issues` link will be displayed on this cookbook's page when
+# uploaded to a Supermarket.
+#
+issues_url 'https://github.com/aashishmodak/cookbooksrepo/SAPDRM-15_hconsul/hconsul/issues'
```
Comment (aashishmodak) [inline hconsul/metadata.rb]: remove this word "SAPDRM-15_hconsul" from url.
This is your branch name. When the PR is merged, branch name will be removed

File: hconsul/metadata.rb [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+
+# The `source_url` points to the development repository for this cookbook.  A
+# `View Source` link will be displayed on this cookbook's page when uploaded to
+# a Supermarket.
+#
+source_url 'https://github.com/aashishmodak/cookbooksrepo/SAPDRM-15_hconsul/hconsul'
```
Comment (aashishmodak) [inline hconsul/metadata.rb]: remove this word "SAPDRM-15_hconsul" from url

File: README.md [hunk 8]
```diff
@@ -1,3 +1,43 @@
...
 This repo is for rean assignments.
+Added Hashicorp Consul cookbook
+
+docker_install Cookbook
+=======================
+Installs Hashicorp Consul from package OR source code to CentOS 7      
+
+Requirements
+------------
+Installs Hashicorp Consul on CentOS 7
```
Comment (aashishmodak) [inline README.md]: You need to update the README for hconsul and not this one

**Comment by aashishmodak** [comment]
"hconsul/files/default/consul_0.9.0_linux_amd64/consul" - this file is still in PR.

**Review by aashishmodak — CHANGES_REQUESTED** [review]
Fix the review comments and merge conflicts

**Comment by RajeshwariMakond** [comment]
conflict merged please review

**Review by aashishmodak — CHANGES_REQUESTED** [review]
Do you need this file 
"hconsul/files/default/consul_0.9.0_linux_amd64/consul"

**Comment by aashishmodak** [comment]
and resolve merge conflict

File: hconsul/nodes/ip-172-31-45-67.us-west-2.compute.internal.json [hunk 9]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (aashishmodak) [inline hconsul/nodes/ip-172-31-45-67.us-west-2.compute.internal.json:1]: why do we need this file

File: hconsul/recipes/nodes/ip-172-31-45-67.us-west-2.compute.internal.json [hunk 10]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (aashishmodak) [inline hconsul/recipes/nodes/ip-172-31-45-67.us-west-2.compute.internal.json:1]: why do we neeed this file?

**Comment by RajeshwariMakond** [comment]
Please review

**Comment by RajeshwariMakond** [comment]
Issue has been fixed, now can see only one file.Please review.

**Pull request closed without merging**